src/main.rs

```diff
@@ -28,8 +28,14 @@ use probe_rs::{
 use probe_rs_cli_util::build_artifact;
 use probe_rs_rtt::{Rtt, ScanRegion};
 
+const CARGO_NAME: &'static str = env!("CARGO_PKG_NAME");
```

Comment:
Why not add those constants to the cli-util lib? :) I think that should work and then we can also add this to cargo-flash? :)

Reply:
does not seem to work for git_version

```
$ git status 
On branch version
nothing to commit, working tree clean

$ git rev-parse --short HEAD
e0104b6

$ cargo run - --version  
probe-rs-cli-util 0.9.0
git commit: v0.5.1-467-g7b33b4b-modified
```

I don't even see where 0.5.1 comes from :/ but both cargo env variable and git_version do the wrong thing or maybe phrased more accurately are used in a way they were not meant to be ;)

We can still add it to cargo flash as well though, just copy pasting :D

Reply:
Yeah sounds good to me :) Do this now, possibly improve later :)